modules/http-api/src/main/java/org/mule/runtime/http/api/exception/ConnectionAlreadyClosedException.java

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright (c) MuleSoft, Inc.  All rights reserved.  http://www.mulesoft.com
+ * The software in this package is published under the terms of the CPAL v1.0
+ * license, a copy of which has been included with this distribution in the
+ * LICENSE.txt file.
+ */
+
+package org.mule.runtime.http.api.exception;
+
+import java.io.IOException;
+
+/**
+ * This Exception is thrown when there is an error that implies losing the connection with the client.
+ * 
+ * @since 4.4.0
+ */
+public class ConnectionAlreadyClosedException extends IOException {
+
+  public ConnectionAlreadyClosedException() {
```

Comment:
is this usage without a cause valid? based on what we chatted on slack i would assume not.

Reply:
It can be used without a cause, because the connector only asks for the exception type.  But besides that, I'm setting the cause in the service, so you are right, I've already removed this constructor.